include/library/spdm_crypt_lib.h

```diff
@@ -25,6 +25,7 @@
 #define LIBSPDM_MAX_AEAD_KEY_SIZE 32
 #define LIBSPDM_MAX_AEAD_IV_SIZE 12
 #define LIBSPDM_MAX_AEAD_TAG_SIZE 16
+#define LIBSPDM_MAX_PSK_HINT_SIZE 128
```

Comment:
`LIBSPDM_MAX_PSK_HINT_SIZE` needs to be defined by the integrator.

Reply:
We already have `LIBSPDM_PSK_MAX_HINT_LENGTH` in spdm_lib_config.h. Why we need `LIBSPDM_MAX_PSK_HINT_SIZE` here???